### PR TITLE
pyuvdata: new package submission

### PR DESCRIPTION
**Pull request by pkgw**
CC @bhazelton.

### CODE REVIEW

File: recipes/pyuvdata/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    - healpy
+    - nose
+    - python-casacore
+  imports:
+    - pyuvdata
+  commands:
+    - nosetests pyuvdata
+
+about:
+  home: https://github.com/HERA-Team/pyvudata
```
Comment (synapticarbors) [inline recipes/pyuvdata/meta.yaml]: url incorrect (transposed the `u` and the `v`). Should be:

https://github.com/HERA-Team/pyuvdata

File: recipes/pyuvdata/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+{% set sha256 = "8ac8cc6467ff1d73814418bd699dad28fd9f278908f75ea038f20f7821e1b1b5" %}
+
+package:
+  name: {{ name }}
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  #git_url: https://github.com/HERA-Team/pyuvdata.git
```
Comment (synapticarbors) [inline recipes/pyuvdata/meta.yaml]: Delete comments

File: recipes/pyuvdata/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+    - scipy
+    - setuptools
+  run:
+    - aipy >=2.1.6
+    - astropy >=1.2
+    - numpy >=1.11
+    - pyephem
+    - python <3
+    - scipy
+    - setuptools
```
Comment (synapticarbors) [inline recipes/pyuvdata/meta.yaml]: Is `setuptools` an actual runtime req? I didn't see `pkg_resources` used anywhere in the source code.

**Comment by pkgw** [comment]
D'oh, got a bit sloppy there!

**Pull request merged**